### PR TITLE
Add ::search-text, ::details-content, ::before::marker and ::after::marker

### PR DESCRIPTION
**Pull request by yisibl**
@SebastianZ PTAL

### CODE REVIEW

File: tests/css-pseudo-4.js [hunk 1]
```diff
@@ -29,6 +29,13 @@ export default {
 			},
 			tests: ['::selection'],
 		},
+		'::search-text': {
+			links: {
+				tr: '#selectordef-search-text',
```
Comment (SebastianZ) [inline tests/css-pseudo-4.js:34]: Until those new features made it into a TR publication, only the draft links should be provided.

File: tests/css-pseudo-4.js [hunk 2]
```diff
@@ -71,6 +76,21 @@ export default {
...
+		'::file-selector-button': {
+			links: {
+				tr: '#file-selector-button-pseudo',
+				dev: '#file-selector-button-pseudo',
+			},
+			tests: ['::file-selector-button'],
+		},
+		'::details-content': {
+			links: {
+				tr: '#details-content-pseudo',
```
Comment (SebastianZ) [inline tests/css-pseudo-4.js:89]: Like above, the TR link should be removed for the time being.

**Review by SebastianZ — CHANGES_REQUESTED** [review]
The changes look fine. Only the two TR links should be removed for now.

File: tests/css-pseudo-4.js (hunk 1 above)
Comment (yisibl) [reply tests/css-pseudo-4.js]: Thanks, done

**Comment by SebastianZ** [comment]
Thank you! Merged.

**Pull request merged**